src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/DataFrameTests.cs

```diff
@@ -226,7 +226,7 @@ public void TestVectorUdf()
         [Fact]
         public void TestDataFrameVectorUdf()
         {
-            Func<PrimitiveDataFrameColumn<int>, ArrowStringDataFrameColumn, ArrowStringDataFrameColumn> udf1Func =
+            Func<Int32DataFrameColumn, ArrowStringDataFrameColumn, ArrowStringDataFrameColumn> udf1Func =
                 (ages, names) =>
                 {

```

Comment:
Will https://github.com/dotnet/spark/pull/277#discussion_r387794908 be addressed as well?

Reply:
Yup. https://github.com/dotnet/corefxlab/pull/2889 is up to address this.